core/models.py

```diff
@@ -143,6 +143,9 @@ def count_descendants(post: "Post") -> int:
 
         return count_descendants(self)
 
+    def get_comments(self: "Post") -> models.QuerySet:
+        return self.children.all()

```

Comment:
Missing is_active filtering. 
Actually, I am thinking we may need manager on GenericModel so we don't throw is active all over the place. 
- set default manager to filter is_active = Tue
- in admin use second manager, `objects_all` or similar name. 
https://docs.djangoproject.com/en/5.0/topics/db/managers/#modifying-a-manager-s-initial-queryset

Thoughts?